packages/test/local-server-stress-tests/src/ddsOperations.ts

```diff
@@ -118,6 +120,76 @@ export const DDSModelOpReducer: AsyncReducer<DDSModelOp, LocalServerStressState>
 };
 
 export const validateConsistencyOfAllDDS = async (clientA: Client, clientB: Client) => {
+	const connectedClients = [clientA, clientB];
```

Comment:
pulling out the synchronize bit seems good, but it should be in a different step from validate. Validate only needs to compare 2 clients, but before validating we need to ensure that a whole set of clients are at the same state (so `connectedClients` should include all the clients in the test that can have submitted ops)

Reply:
changes look good other than this